spec/spec_helper_acceptance.rb

```diff
@@ -14,6 +14,8 @@
     on host, 'yum clean expire-cache'
   end
 
+  on host, 'export LANG=en_us.UTF-8'
```

Comment:
This will be reverted by a modulesync. Is this really needed?

Reply:
That is what I am trying to figure out in part. I have seen errors related to locale when testing and I want to see if it makes a difference locally and on Travis.